cl8-web/server/scripts/generateBootstrapUser.js

```diff
@@ -33,6 +36,79 @@ const admin = setupFbaseAdmin()
 //     }
 //   }]
 
+// Function to import from CSV. Requires a file path
+const importCsv = (file) => {
+  // Declare array to hold all rows
+  let data = []
+
+  // Open a file stream to read rows
+  fs.createReadStream(file)
+      .pipe(
+          parse({
+            columns: true,
+            cast: (value, context) => {
+              if (value.length > 0) {
+                if (value.toLowerCase() === 'yes') {
+                  return true
+                } else if (value.toLowerCase() === 'no') {
+                  return false
+                } else {
+                  return value
+                }
+              } else {
+                return null
+              }
+            },
+          })
+      )
+      .on('data', (csvrow) => {
+        data.push(csvrow)
+      })
+      .on('end', () => {
+        data.forEach((user) => {
+          // Run cleanups on each individual user
+          if (user.tags) {
+            fixNested(user)
+          }
+          clean(user)
+        })
+
+        // Return the array
+        return data
+      })
+}
+
+// This is used to apply logic on the nested fields.
+const fixNested = (user) => {
+  if (user.tags) {
+    const tags = user.tags.split(',')
+    const tagList = []
+    tags.forEach((tag) => {
+      const code = Math.random().toString(36).substring(2, 11);
+      const id = 'tempval' + code
+      tagList.push({
+        code: code,
+        id: id,
+        name: tag
+      })
+    })
+    user.tags = tagList
+  }
+  if (user.photo) {
```

Comment:
This is a really nice touch Ryan 👍 

I think it'll makes it easier to add the corresponding photos for users too in the import.